### ISSUE TITLE
Version of numpy that satisfies tensorflow and gputools?

### ISSUE THREAD

**Issue by Cocomolch4000** [stardist/stardist#143]
Hi there,

I was trying to run stardist on gpu. but "gputools" throws errors for all numpy versions that are compatible with tensorflow (1.19.2 - 1.19.5), are there any workarounds? for older or newer numpy versions gputools seems to work.

Best
Cocomolch

**Comment by maweigert** [comment]
> but "gputools" throws errors for all numpy versions 

What kind of error?

> that are compatible with tensorflow (1.19.2 - 1.19.5)

tensorflow is compatible with recent numpy version (e.g. 1.20.x), why not using this?

**Comment by Cocomolch4000** [comment]
The error was "ImportError: ImportError: numpy.core.multiarray failed to import". 
Sorry I was a bit in a hurry yesterday, pip told me that even with versions 1.20.x of numpy tensorflow would not work "tensorflow 2.4.0 requires numpy~=1.19.2, but you have numpy 1.20.3 which is incompatible.".

I ignored this and stardist and tensorflow just work, however, although my gputools_available() outputs "True" and I set use_gpu to "True" only my CPU is busy, GPU remains at 0-1% workload during training. I am at 3rd epoch right now.
Am I forgetting something? Sorry for constant bothering..
